pages/index.md

```diff
@@ -1,22 +1,37 @@
 # CCR Research Computing Documentation
 
-CCR is a high-performance computing center offering faculty and students access
-to supercomputing environments, high-end visualization services, an on-premise
-research cloud, and experienced staff to help you move your research forward.
+The [Center for Computational Research (CCR)](https://buffalo.edu/ccr) at the [University at Buffalo](https://buffalo.edu) is a high-performance research computing center offering faculty, staff, students, and local businesses access to supercomputing environments, high-end visualization services, an on-premise research cloud infrastructure, and experienced staff to help you move your research forward.  
 
-This documentation covers the use of CCR's research computing resources. Not
-sure where to start? Try some of these links to get you started:
+This documentation covers the use of CCR's research computing and research cloud resources. More general information about CCR can be found on [our website](https://buffalo.edu/ccr).  Requests for help can be emailed to ccr-help_at_buffalo.edu or submitted via our [Freshdesk help desk portal](https://ubccr.freshdesk.com).   
 
-- [Logging In](hpc/login.md)
-- [Research Computing Filesystems](hpc/storage.md)
-- [Compiling Software](software/building.md)
-- [Batch Jobs](hpc/jobs.md)
-- [The Module System](software/modules.md)
-- [Frequently Asked Questions (FAQ)](faq.md)
+!!! Tip   
+    CCR provides UB's research computing and research cloud resources. All other IT services and support at UB are provided by [UBIT](https://buffalo.edu/ubit)  
 
-!!! note "Can't find what you need?"
-    Provide feedback on our github repo! 
+**Not sure where to start?** These links provide info on the most popular topics while the left navigation provides the full documentation organized by topics.  You can also use the search box at the top left for context searching.  
+
+- [Getting Access](getting-access.md)  
+- [Getting Started - Once you Have Access](getting-started.md)  
+- [Frequently Asked Questions (FAQ)](faq.md)  
+- [Resources Provided by CCR](getting-started.md#computing-resources-at-ccr)
+- **Logging in to:**
+    - [HPC clusters via SSH](hpc/login.md)  
+    - [HPC clusters via OnDemand](portals/ood.md)  
+    - [Cloud dashboard](cloud/using.md)  
+    - Other portals & services: see portal menu on the left  
+- **Research Computing (HPC) Specific Topics:**  
+    - [Batch Job System](hpc/jobs.md)  
+    - [Data Transfer Options](hpc/data-transfer.md)  
+    - [Linux and Slurm Cheat Sheet](https://buffalo.box.com/s/nqj3neyt2w1dtb3gix6zxqx5gcc9x30n)  
+    - [Software Module System](software/modules.md)  
+    - [Compiling Software](software/building.md)  
+    - [Visualization Servers](hpc/viz.md)  
+- **Research Cloud Specific Topics:**  
+    - [LakeEffect Cloud Quick Start](cloud/lake-effect)  
```

Comment:
These links need fixing.. missing *.md  should be `cloud/lake-effect.md` .. etc.